.github/workflows/main.yml

```diff
@@ -11,6 +11,6 @@ jobs:
       with:
         fetch-depth: '0'
     - name: Bump version and push tag
-      uses: anothrNick/github-tag-action@master
+      uses: geoadmin/github-tag-action@master
       env:
```

Comment:
?

Reply:
Oops sorry I pushed a commit meant to my fork only on the wrong branch, I reverted it